test/models/content_credential_test.rb

```diff
@@ -17,41 +17,41 @@ module Katello
 
       it "should be successful with valid parameters" do
         content_credential = ContentCredential.new(:name => "Gpg Key 1", :content => @test_gpg_content, :organization => @organization)
-        content_credential.must_be :valid?
+        _(content_credential).must_be :valid?
       end
 
       it 'should be destroyable' do
         content_credential = ContentCredential.create!(:name => "Gpg Key 1", :content => @test_gpg_content, :organization => @organization)
         product = katello_products(:fedora)
         product.gpg_key = content_credential
         product.save!
-        content_credential.destroy.wont_equal(false)
+        _(content_credential.destroy).wont_equal(false)
       end
 
       it "should be unsuccessful without content" do
         content_credential = ContentCredential.new(:name => "Gpg Key 1", :organization => @organization)
-        content_credential.wont_be :valid?
+        _(content_credential).wont_be :valid?
       end
 
       it "should be unsuccessful without a name" do
         content_credential = ContentCredential.new(:content => @test_gpg_content, :organization => @organization)
-        content_credential.wont_be :valid?
+        _(content_credential).wont_be :valid?
       end
 
       it "should be unsuccessful without proper gpg key" do
         content_credential = ContentCredential.new(:name => "Gpg Key 1", :content => "foo-bar-baz", :organization => @organization)
         if SETTINGS[:katello][:gpg_strict_validation]
-          content_credential.wont_be :valid?
+          _(content_credential).wont_be :valid?
         else
-          content_credential.must_be :valid?
+          _(content_credential).must_be :valid?
         end
       end
 
       it "should be unsuccessful with binary content" do
         content = "\x81\xA4user\x83\xA3age\x18\xA4name\xA4ivan\xA5float\xCB@\x93J=p\xA3\xD7\n"
         content.force_encoding(::Encoding::ASCII_8BIT)
         content_credential = ContentCredential.new(:name => "Gpg Key 1", :content => content, :organization => @organization)
-        content_credential.wont_be :valid?
+        _(content_credential).wont_be :valid?
```

Comment:
I am not exactly what this is doing I thought `_()` is what we use for i18n
Looking at https://docs.rubocop.org/rubocop-minitest/cops_minitest.html#minitestglobalexpectations I am going to suggest  adding a `EnforcedStyle: expect`
of the 3 styles listed in docs I like this the best
```ruby
# bad
musts.must_equal expected_musts
wonts.wont_match expected_wonts
musts.must_raise TypeError

_(musts).must_equal expected_musts
_(wonts).wont_match expected_wonts
_ { musts }.must_raise TypeError

value(musts).must_equal expected_musts
value(wonts).wont_match expected_wonts
value { musts }.must_raise TypeError

# good
expect(musts).must_equal expected_musts
expect(wonts).wont_match expected_wonts
expect { musts }.must_raise TypeErro
```

Reply:
As per our discussion, `EnforcedStyle` doesn't work.

I found out that `EnforcedStyle` was added to the Minitest/GlobalExpectations cop in `rubocop-minitest` version https://github.com/rubocop/rubocop-minitest/releases/tag/v0.16.0, just 23 days ago.

Problem is, upgrading would require 

1. Upgrading `rubocop-minitest` in `theforeman-rubocop` [here](https://github.com/theforeman/theforeman-rubocop/blob/fc0ef342248f189750a1295e78628b56fbe4eb25/theforeman-rubocop.gemspec#L26)
2. Release a new version of `theforeman-rubocop` gem (hi @ezr-ondrej :)
2. Then, Foreman would need to update its version of `theforeman-rubocop` that it consumes [here](https://github.com/theforeman/foreman/blob/e98fc0116a9a5bb80a2f1a15139f813943827240/bundler.d/test.rb#L22)
3. Finally, Katello could start using `EnforcedStyle`.

(This is one of the reasons I hate prescriptive wrappers for developer tooling.)
The above seems like too much work.  Also, manually fixing 147 occurrences seems like too much work.  So I am proposing we either
1. keep the rule disabled; or
1. become okay with the `_()` style of usage.

Thoughts?

Reply:
am i reading it correctly that `_()` and `_ {}` are being added because we are using deprecated expects?  

I feel like we should probably update those to use the non-deprecated methods?  (Or am i misunderstanding something)

Reply:
@jlsherrill That's right, but unfortunately auto-correct won't help us so it's going to take some time. If you think it's worth it I guess I can do it..

Reply:
I think its something we should do as 1) i presume those methods will go away at some point and 2) so no one adds more via copy/paste

We can do it as a 2nd task if we want to move that one action to another card.  If we do that i'd say we should disable that cop for now rather than implement the _() nomenclature.

Reply:
I suggest moving the later part to another card @jeremylenz . I don't find this cop as useful.

Reply:
Disabled the rule and created follow-up card. :+1:

Reply:
I'm -1 to a follow up card. I perceive the redmine this is being worked from as already being a follow-on card. And adding a follow-on to a follow-on feels like kicking the can down the road. Is it really so bad to fix a few dozen expectations?

Reply:
> Problem is, upgrading would require
> 
> 1. Upgrading `rubocop-minitest` in `theforeman-rubocop` [here](https://github.com/theforeman/theforeman-rubocop/blob/fc0ef342248f189750a1295e78628b56fbe4eb25/theforeman-rubocop.gemspec#L26)
> 2. Release a new version of `theforeman-rubocop` gem (hi @ezr-ondrej :)
> 3. Then, Foreman would need to update its version of `theforeman-rubocop` that it consumes [here](https://github.com/theforeman/foreman/blob/e98fc0116a9a5bb80a2f1a15139f813943827240/bundler.d/test.rb#L22)
> 4. Finally, Katello could start using `EnforcedStyle`.

So I've done 1. and 2. but I do not believe you need 3. I've also put together a [GH action doc](https://github.com/theforeman/theforeman-rubocop/blob/main/docs/github_action.md) where I explain the main point that you should run rubocop from the plugin folder. The weird adding rake task to foreman and than run the rake task was always a bad idea. That way a plugin can depend on any version of `theforeman-rubocop` it wants :shrug: :)

Reply:
> (This is one of the reasons I hate prescriptive wrappers for developer tooling.) The above seems like too much work. Also, manually fixing 147 occurrences seems like too much work. So I am proposing we either
> 
> 1. keep the rule disabled; or
> 2. become okay with the `_()` style of usage.
> 
> Thoughts?

I can't stop myself but meddle once more:

The `_()` is quite fine and I find it quite appealing as it shows more clearly the subject of the test.
The reasoning behind using that syntax over directly calling the `must_` methods on objects is clear: you do not want to have `must_` methods defined directly on a `Object` as it pollutes it quite a bit and it can interfere with some methods you define.